doc/swish/erlang.tex

```diff
@@ -88,8 +88,9 @@
 A \emph{process} is an instance of \code{pcb}, a Scheme record type
 that extends \code{q} with an immutable \code{id} field, the process's
 unique positive exact integer, an immutable \code{create-time} field,
-the process's create time from \code{erlang:now}, and the following
-mutable fields:
+the process's create time from \code{erlang:now}, an immutable
+\code{parameters} field, the process's weak eq-hashtable of process
```

Comment:
Consider "... weak eq-hashtable **mapping** process parameters to values ..." which is more like what we had at line 1109 before.

Reply:
Good suggestion, thanks!